plugins/modules/apt_rpm.py

```diff
@@ -165,6 +167,21 @@ def query_package(module, name):
         return False
 
 
+def check_package_version(module, name):
+    # compare installed and candidate version
+    # if newest version already installed return True
+    # otherwise return False
+
+    rc, out, err = module.run_command("%s policy %s" % (APT_CACHE, name), \
```

Comment:
It is better to use the list version of `run_command()` than the string version:
```suggestion
    rc, out, err = module.run_command([APT_CACHE, "policy", name], \
```